src/entitysdk/downloaders/emodel.py

```diff
@@ -0,0 +1,22 @@
+"""Download functions for EModel entities."""
+
+from pathlib import Path
+
+
+def download_hoc(client, emodel, hoc_dir="./hoc"):
+    """Download hoc file.
+
+    Args:
+        client (Client): EntitySDK client
+        emodel (EModel): EModel entitysdk object
+        hoc_dir (str or Pathlib.Path): directory to save the hoc file
+    """
+    hoc_dir = Path(hoc_dir)
+    hoc_dir.mkdir(parents=True, exist_ok=True)
```

Comment:
Is this needed? download_asset_file creates intermediate directories iirc, I may be wrong though. Could you please check?